test/jobs/good_job_statsd_job_test.rb

```diff
@@ -3,7 +3,7 @@
 class GoodJobStatsDJobTest < ActiveSupport::TestCase
   include StatsD::Instrument::Assertions
 
-  class FailureJob < ApplicationJob
+  class FailureJob < ActiveJob::Base # opt-out default retries
```

Comment:
I have fixed tests by opting out of default this way @segiddins. Is that ok?

Reply:
yup that works, thank you!